index.html

```diff
@@ -9,4 +10,355 @@
+                            </div>
+                            <h7> Electronics </h7>
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (1).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Home </h7>
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (3).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Aplliances </h7>
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (4).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Travel </h7>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (6).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Toys</h7>
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <hr>
+        <!--Complete Deal Section-->
+
+
+        <!----Deals Of the Day-->
+        <section>
+            <div class="container">
+                <hr class="light">
+                <span class="text"> Deals Of the Day </span>
+                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/images.png" class="clock" height="2%"
+                    width="2%">
+                <span class="time"> 22:07:36 Left </span>
+                <button class="bk-light bg-primary">VIEW ALL</button>
```

Comment:
@Akshay8122 VIEW ALL button doesn't look as per the design. Check and fix.
![Screenshot 2022-01-13 at 9 50 09 AM](https://user-images.githubusercontent.com/75989919/149265367-d339c051-2b57-43f7-a517-5dbb991e93f8.png)